packages/ui/src/components/project/ProjectHeader.vue

```diff
@@ -37,7 +37,11 @@
       <div v-if="project.categories.length > 0" class="hidden items-center gap-2 md:flex">
         <TagsIcon class="h-6 w-6 text-secondary" />
         <div class="flex flex-wrap gap-2">
-          <TagItem v-for="(category, index) in project.categories" :key="index">
+          <TagItem
+            v-for="(category, index) in project.categories"
+            :key="index"
+            :action="() => router.push(`../${project.project_type}s?f=categories:${category}`)"
```

Comment:
The `../` seems fragile and would probably not work on child pages, I would use an absolute path from the root by just removing the `..` part. 

Otherwise, I think this works. I may improve the implementation of the routing to use an actual link in the future, but for now I think this is fine as it'll be easier to implement with some changes on another branch.